### ISSUE TITLE
[PlotWidget]Marker is initially not drawn with mpl 1.5.0--2.0.2

### ISSUE THREAD

**Issue by PiRK** [silx-kit/silx#1347]
When adding a marker, it is initially not visible.
It becomes visible when some interaction happens (zoom, pan, marker dragged, curve clicked).

```
from silx.gui import qt
from silx.gui.plot import PlotWidget
app = qt.QApplication([])
plot = PlotWidget()

plot.addCurve(x=[0, 2], y=[10, 20])
plot.addXMarker(0.85, legend="a",
                            text="toto", #color="blue",
                            draggable=True)
plot.show()
app.exec_()
```

**Comment by t20100** [comment]
It is due to rendering markers as "overlays" but no idea so far why this is not working for the first display of the widget....

**Comment by t20100** [comment]
closed by #1984
